mealy/tests/test_error_analyzer.py

```diff
@@ -29,7 +29,7 @@ def test_with_only_scikit_model():
     rf = RandomForestClassifier()
     rf.fit(X_train, y_train)
     mpp = ErrorAnalyzer(rf, feature_names=numeric_features)
-    mpp.fit(X_test, y_test)
+    mpp.fit(X_test, y_test, tree_params={'max_depth': [5, 10, None], 'min_samples_leaf': [10, 20]})
```

Comment:
Maybe I am missing something obvious, should this not be passed as the `param_grid` parameter of  `ErrorAnalyzer`? Same question for l.92

Reply:
good catch, fixed in f2090d3